gel/_internal/_qbmodel/_pydantic/_models.py

```diff
@@ -1197,15 +1177,15 @@ class _MergedModelBase(GelModel, metaclass=_MergedModelMeta):
 
 class ProxyModel(
     GelModel,
-    Generic[_MT_co],
+    _abstract.AbstractGelProxyModel[_MT_co, GelLinkModel],
     __gel_root_class__=True,
 ):
     __gel_has_id_field__ = False
 
     if TYPE_CHECKING:
         __gel_proxy_merged_model_cache__: ClassVar[type[pydantic.BaseModel]]
 
-    __slots__ = ("__linkprops__", "_p__obj__")
```

Comment:
Interesting. I guess this can fix serialization with exclude/include filters? why did you do this?

Reply:
Slots are incompatible with descriptors.